### ISSUE TITLE
Didn't find any other way to contact 

### ISSUE THREAD

**Issue by Tejashree21** [AzureTeachNet/AzureTeachTutorials#1]
Can you upload code for  
https://youtube.com/playlist?list=PLW6IC4y81fVvQj4B29R6HzYgxyBl6JdbH

&

https://youtube.com/playlist?list=PLW6IC4y81fVtmVYkccLTrSeJXslvn71st     ?

And also please put links of videos for better understanding.

**Comment by AzureTeachNet** [comment]
Thanks for your feedback. As per the youtube rules, I should not watch my own videos.

could you please let me know the titles of the videos?

**Comment by Tejashree21** [comment]
Web API Tutorials 
1. Web API Download File c# | Download CSV File | Data table to CSV | Web API Return File
2. Upload File Asp.net Core Web API
 3. Asp.net Core Authentication With JWT(Json Web Token) & Refresh Tokens
4. Refresh Token & Jwt Token
5. Web Api Model validation | .Net 5 | Asp.net Core
6. .Net Core Web API Azure AD Authentication and Authorization
7. Web API token based authentication example C# step by step | Asp.Net Core
8. Angular Web API Download File | Complete Tutorial
9. Net6 Minimal Api Upload File | Asp net Core 6 Minimal API File Upload
10. Minimal api dotnet 6 | .net 6 Minimal Api Download File
11. How to Create an API with Minimal APIs | Minimal API upload file
12. Web Api Upload File|Upload File To Azure Storage Account
13. Web Api+Azure Storage Account Download File|Download File from Azure storage
14. Web Api Download Multiple Files


Azure Storage Account Tutorials
1. Azure Storage Account Tutorial - Overview
2. Azure Blob Storage -Access Blob From Azure Storage Account Using Postman Tool
3. Azure Storage Account - Azure Blob Storage Snapshots
4. Blob Versioning | azure storage account tutorial
5. Download All Blobs From Container - Azure Storage Account - .Net Core
6. Angular App Azure Deployment | Azure Devops | Azure Static Web Hosting
7. Read Blob - Azure Storage Account - .Net Core
8. Delete All Files From Storage Account Container - Azure Blob Storage Account
9. Delete Blob From Container - Azure Storage Account - .Net Core
10. Upload File To Azure Blob Storage | Azure Storage Account | .Net Core
11. Upload files to Microsoft Azure Blob storage with C# | Azure Storage Account | .Net Core
12. Modify File in Storage Account | Upload Blobs to Storage Account | Azure Storage Account | .Net Core
13. What is the use of Azure Blob Leases? | Upload Blobs To Azure | Azure Storage Account
14. Securely Access File | Azure Storage Account | .Net core
15. Azure Storage Account | Blob Hot vs Cool vs Archive | Optimize Costs with Azure
16. Web Api+Azure Storage Account Download File|Download File from Azure storage
17. Azure SAS | Azure Shared Access Signature | Storage Account Shared Access


Basically it will helpful if you create YouTube playlist wise folders here.
